npm/private/transitive_closure.bzl

```diff
@@ -45,14 +46,22 @@ def gather_transitive_closure(packages, package, no_optional):
             transitive_closure[name] = transitive_closure.get(name, [])
             if version in transitive_closure[name]:
                 continue
-            transitive_closure[name].insert(0, version)
+            transitive_closure[name].append(version)
```

Comment:
These get sorted before returning so the insertion-order doesn't matter. I assume appending is more optimal then inserting. Was there any (known) reason for inserting at the start?

Reply:
I don't recall